lua/cmake/init.lua

```diff
@@ -61,24 +61,30 @@ function cmake.debug(...)
   end
 
   -- Split on spaces unless "in quotes"
-  local splitted_args = vim.fn.split(arguments, [[\s\%(\%([^'"]*\(['"]\)[^'"]*\1\)*[^'"]*$\)\@=]])
+  local splitted_args
+  if arguments then
+    splitted_args = vim.fn.split(arguments, [[\s\%(\%([^'"]*\(['"]\)[^'"]*\1\)*[^'"]*$\)\@=]])
+  else
+    splitted_args = {}
+  end
 
   -- Remove quotes
   for i, arg in ipairs(splitted_args) do
     splitted_args[i] = arg:gsub('"', ''):gsub("'", '')
   end
 
-  table.insert(splitted_args, { ... })
+  vim.list_extend(splitted_args, { ... })
 
   vim.api.nvim_command('cclose')
   local config = {
-    type = 'cpp',
-    name = 'Debug CMake target',
-    request = 'launch',
+    name = 'Debug ' .. parameters['currentTarget'],
```

Comment:
Could you remove `Debug` here. I know that it was like this before, but I think it would be better without it :)

```suggestion
    name = parameters['currentTarget'],
```

Reply:
>  update options table

You mean list the default configuration in `README.md`?

> mention that by default it configured for lldb-vscode?

Also in `README.md`?

> Could you remove Debug from the name?

OK.

Reply:
Or you mean change the default value of `type` key to `lldb-vscode`, or just mention it in code comments?

Reply:
Done.